### ISSUE TITLE
Should I use CSS or JS? 

### ISSUE THREAD

**Issue by adaptifyDesigns** [flackr/scroll-timeline#150]
For a production site does it make sense to use CSS for all the view timeline scroll animations? Does the latest polyfill actually cover my ass and provide a fallback for incompatible browsers? Or should I keep using JS for these animations?

**Comment by kevers-google** [comment]
For the most part, the choice of CSS or JavaScript is a developer preference.  In the polyfill, there is an additional abstraction layer with CSS since we need to parse the stylesheets to extract properties that are not natively supported.  There are some relatively nuanced and unavoidable timing discrepancies, as some synchronous updates need to be asynchronous in the polyfill.  The polyfill aims to provide both missing JavaScript API support as well as missing CSS support.  At the moment, CSS support still has a few holes such as not supporting the latest changes to attachment scope rules, and not fully supporting inline style changes.  Working on plugging those holes, but cannot give an ETA.

In short, I cannot offer a definitive recommendation of JavaScript vs CSS since it depends largely on external factors such as ease of development, animation/CSS reuse, etc. If looking for the path that is likely to have less code churn as we make the polyfill fully spec compliant (modulo minor timing discrepancies), then JavaScript is the safer bet.

Scroll-driven animations are fully supported in the latest Chrome release in alignment with the scroll-animations-1 spec (https://www.w3.org/TR/scroll-animations-1/).  One bit of advice would be to test pages with Chrome and if it works in Chrome but not with an unsupported-browser + polyfill then please let us know.

**Comment by bramus** [comment]
Closing this issue as the question has been answered.
